packages/venia-concept/src/components/MiniCart/miniCart.css

```diff
@@ -1,7 +1,9 @@
 .root {
     --base-z-index: 4;
-    --minicart-height: 100vh;
```

Comment:
Having trouble finding our supported browser matrix but for browsers that cannot support custom props we should still have `height: 100vh`. See [the example](https://css-tricks.com/the-trick-to-viewport-units-on-mobile/#article-header-id-1).

Reply:
Resolved in https://github.com/magento-research/pwa-studio/pull/1387/commits/fd9fa4d5173a2660786c410dcebab0f4ce27f095.